spec.html

```diff
@@ -36466,8 +36479,7 @@ <h1>
           <dl class="header">
           </dl>
           <emu-alg>
-            1. Assert: _match_.[[StartIndex]] is a non-negative integer less than or equal to the length of _S_.
-            1. Assert: _match_.[[EndIndex]] is an integer between _match_.[[StartIndex]] and the length of _S_, inclusive.
+            1. Assert: _match_.[[StartIndex]] &le; _match_.[[EndIndex]] &le; the length of _S_.
```

Comment:
That loses the assertion that `_match_.[[StartIndex]]` is a non-negative integer. So:
```suggestion
            1. Assert: 0 &le; _match_.[[StartIndex]] &le; _match_.[[EndIndex]] &le; the length of _S_.
```

And likewise for the next diff.

Reply:
It also no longer says anything about it being integral. That was intentional. `[[StartIndex]]` is [declared to be a non-negative integer](https://tc39.es/ecma262/#table-match-record).